tonic-health/Cargo.toml

```diff
@@ -21,11 +19,16 @@ transport = []
 [dependencies]
 async-stream = "0.3"
 prost = "0.13"
-tokio = {version = "1.0", features = ["sync"]}
+tokio = { version = "1.0", features = ["sync"] }
```

Comment:
While I like this change and believe it is broadly idiomatic in matching the ecosystem at large, I believe the other `Cargo.toml` in this repo do not follow this style. If we're going to do this, I'd prefer to do it in a separate PR (or a separate commit) that does this consistently across all `Cargo.toml` files in the repo.